content/working_with/console/widgets/filters.md

```diff
@@ -40,9 +40,28 @@ Add, edit and clone operation modals share a common component for defining filte
 The component presents a list of rows, each representing a single filter rule. Each row contains three inputs:
 
 * Rule type selection dropdown - selecting the context of the rule which can be based on labels or supported deployment attributes such as blueprint ID, site name, or creator.
-* Rule operator dropdown. The set of available operators to choose from depends on the selected rule type.
+* Rule operator dropdown. The set of available operators to choose from depends on the selected rule type. See Table 1. and Table 2. below for details.
 * Value input (for attribute rules) or key/value input(s) (for label rules). 
 
+| Operator label in the UI       | Operator name in the API/CLI        |
```

Comment:
I think we should also include the logic. Here is my suggestion for the tables:

| Operator name in the UI | Operator name in the API/CLI | Applied logic |
|:------------------------------:|:-----------------------------------:|:--------:|
| is one of                      | `any_of`                            | The label key matches the specified key and the label value matches one of the specified values. |
| is not one of                  | `not_any_of`                        | The label key matches the specified key and the label value does not match any of the specified values. |
| is not one of (or no such key) | `is_not`                            | No label key matches the specified key, or the label key matches the specified key and the label value does not match any of the specified values. |
| key is not                     | `is_null`                           | No label key matches the specified key. |
| key is                         | `is_not_null`                       | The label key matches the specified key. |
**Table 1**. Labels operators

| Operator name in the UI  | Operator name in the API/CLI        | Applied logic |
|:-------------------------:|:-----------------------------------:|:------------:|
|   is one of               | `any_of`                            | The deployment property matches one of the specified values. | 
|   is not one of           | `not_any_of`                        | The deployment property does not match any of the specified values. |
|   contains                | `contains`                          | The deployment property contains the specified value. |
|   does not contain        | `not_contains`                      | The deployment property does not contain the specified value. |
|   starts with             | `starts_with`                       | The deployment property starts with the specified value. |
|   ends with               | `ends_with`                         | The deployment property ends with the specified value. |
**Table 2**. Attributes (Blueprint, Site name, Creator) operators mapping

Reply:
Thanks for the detailed description. I have applied it with some changes (`deployment property` -> `deployment attribute`).